ledfx/integrations/mood_manager.py

```
@@ -0,0 +1,1194 @@
+                if section_changed and section is not None:
+                    self._last_section = section
+
+            if section_changed and section is not None:
+                _LOGGER.info(f"Section changed: {section.value}")
+
+                # Fire structure change event
+                try:
+                    self._ledfx.events.fire_event(
+                        StructureChangeEvent(section, event)
+                    )
+                except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)